drivers/input/input_gpio_keys.c

```diff
@@ -137,7 +137,7 @@ static int gpio_keys_interrupt_configure(const struct gpio_dt_spec *gpio_spec,
 		return ret;
 	}
 
-	cb->pin_state = -1;
+	cb->pin_state = gpio_pin_get_dt(gpio_spec);
```

Comment:
Hey could you change the `gpio_ping_get` at line 65 with `gpio_pin_get_dt` as well while you are at it? Sorry I don't normally ask for unrelated changes but this is simple enough and you have a compliance error so this needs to be updated anyway :-)

Feel free to do it in the same or on a separate commit.